dbtwiz/main.py

```diff
@@ -20,7 +21,10 @@ class InvalidArgumentsError(ValueError):
 
 
 # Add admin commands as subcommands of 'admin'
-app.add_typer(admin.app, name="admin")
+app.add_typer(admin.app, name="admin", help="Administrative commands")
+
+# Add generator commands as subcommands of 'generate'
+app.add_typer(generate.app, name="generate", help="Generator commands")
```

Comment:
I think "Generator commands" should be changed to something more explicit like "Create new dbt model or source".

Reply:
Seeing that it isn't enough to use `generate` by itself, `help` should also say something about this (that you must write either `generate model` or `generate source`).

I'm unsure whether having `generate model` and `generate source` is better than having `generate --type model` and `generate --type source`.
For the latter we could have `model` as default, which is something I expect most users to use.

A third alternative could be to be very explicit, and have `generate-model` and `generate-source` as the options.

![image](https://github.com/user-attachments/assets/3a95c523-c245-4570-8ca4-7eda4beb6534)

Reply:
We should discuss this and the previous point with the team, I think.

It's no big deal for me how we name them, I will probably just make some personal macros `dgm` and `dgs` anyway. :)

Initially I made them separate commands due to intending to allow specifying most model/source attributes as command options instead of being asked, but reconsidered that for now.

Reply:
I'll probably also add a poetry alias to be used here.

Reply:
Adjusted the helper texts to be more user friendly

Reply:
Renamed from `generate` to `create`